tests/unit/test_utils/test_vdb/test_vdb_base.py

```
@@ -0,0 +1,188 @@
+        return {"status": "healthy"}
+
+    def create_collection(
+        self,
+        collection_name: str,
+        dimension: int = 2048,
+        collection_type: str = "text",
+    ) -> None:
+        """Create a new collection."""
+        pass
+
+    def check_collection_exists(self, collection_name: str) -> bool:
```

Ruff (0.14.8)

Unused method argument: `collection_name`

(ARG002)